sos/policies/__init__.py

```diff
@@ -800,17 +801,11 @@ def get_preferred_hash_name(self):
         if self._preferred_hash_name:
             return self._preferred_hash_name
 
-        checksum = "md5"
-        try:
-            fp = open("/proc/sys/crypto/fips_enabled", "r")
-        except IOError:
-            self._preferred_hash_name = checksum
-            return checksum
-
-        fips_enabled = fp.read()
-        if fips_enabled.find("1") >= 0:
+        if sys.version_info[0] <= 2:
+            checksum = "md5"
+        else:
```

Comment:
This would break RHEL7 which has a py2 runtime and an optional FIPS mode. You can't just assume that Python 2 equates to a non-FIPS environment.